src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -59,7 +34,22 @@ public Credentials(String credentialsPath) {
      * @param authEndpoint Auth endpoint
      * @param apiEndpoint Domain endpoint of the api, e.g. https://<prefix>.api.lucidtech.ai/<version>
      */
-    public Credentials(String clientId, String clientSecret, String apiKey, String authEndpoint, String apiEndpoint) {
+    public Credentials(
+        String clientId,
+        String clientSecret,
+        String apiKey,
+        String authEndpoint,
+        String apiEndpoint
+    ) throws MissingCredentialsException {
+        if (apiEndpoint == null
+            || authEndpoint == null
+            || apiKey == null
+            || clientSecret == null
+            || clientId == null
+        ) {
+            throw new MissingCredentialsException();
```

Comment:
Would it look better to implement a function that checks a list of variables for null? (Like the "all" function you had implemented earlier)